docs/api/routing.rst

```diff
@@ -374,6 +374,54 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other architectures. Consider the task of building an API 
+for a calculator which can both add add subtract two numbers. You could implement the following:
```

Comment:
_add add subtract_ (sic) :arrow_right: _add and subtract_

Reply:
Ah, missed that, fixed